src/RealtimeEffectPanel.cpp

```diff
@@ -445,7 +445,7 @@ namespace
          // if play is in progress
          bool changed = effectPlugin->ShowHostInterface(
             ProjectWindow::Get( *mProject), DialogFactory(mEffectState),
-            pInstance, *access, true );
+            pInstance, *access, false );
```

Comment:
Maybe the commit that does this should logically be last in the sequence, because all else is in place not to cause bugs in non-modality only then

Reply:
Agreed!

Reply:
Turned out that this commit is not needed at all